BasePolicies/CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Node.js Community Contributing Guide 1.0
+
+This document describes a very simple process suitable for most projects
+in the Node.js ecosystem. Projects are encouraged to adopt this whether they
+are hosted in the Node.js or not.
```

Comment:
That should say "Foundation" :) We host projects in the foundation that aren't in the github org.